topics/imaging/tutorials/imaging-introduction/tutorial.md

```diff
@@ -0,0 +1,206 @@
+
+> ### {% icon question %} Questions
+>
+> You can observe that the image content is barely visible. Why?
+>
+> > ### {% icon solution %} Solution
+> > The original image is 16-bit and the intensity values are spread over a larger range than the
+> > display can render. Therefore, for improved visibility the intensity histogram of the image can
+> > be normalized first.
+> {: .solution }
+{: .question}
+
+
+Next we will normalize the histogram.
+
+> ### {% icon hands_on %} Hands-on: Normalize Histogram and Convert Image
+>
+> 1. **Histogram equalization** {% icon tool %} with the following parameters to normalize the histogram of the image:
+>    - {% icon param-file %} *"Source file"*: `input.tif` file
+>    - *"Histogram Equalization Algorithm"*: `CLAHE`
+> 2. Rename {% icon galaxy-pencil %} the generated file to `input_normalized`
+> 3. **Convert image** {% icon tool %} with the following parameters to convert the image to PNG:
+>    - {% icon param-file %} *"Input Image"*: `input_normalized` file (output of **Histogram equalization** {% icon tool %})
+>    - *"Output data type"*: `PNG`
+> 4. Rename {% icon galaxy-pencil %} the generated file to `viz_normalized`
+> 5. Click on the {% icon galaxy-eye %} (eye) icon next to the file name, to look at the file content
+{: .hands_on}
+
+Your image should now look something like this:
+
+![](../../images/imaging-introduction/viz_normalized.png){: width="75%"}
+
+We can now clearly make out the presence of the stained nuclei. Next we will automatically detect these features and segment the image.
+
+# Image Filtering
+
+Specific features of interest (e.g., edges, noise) can be enhanced or suppressed by using an image filter.
+
+> ### {% icon hands_on %} Hands-on: Filter image
+>
+> 1. **Filter Image** {% icon tool %} with the following parameters to smooth the image:
+>    - *"Image type"*: `Gaussian Blur`
+>    - *"Radius/Sigma"*: `3`
+>    - {% icon param-file %} *"Source file"*: `input.tif` file
+> 2. Rename {% icon galaxy-pencil %} the generated file to `input_smoothed`
+> 3. **Histogram equalization** {% icon tool %} with the following parameters to normalize the histogram of the image:
+>    - {% icon param-file %} *"Source file"*: `input_smoothed` file (output of **Filter image** {% icon tool %})
+>    - *"Histogram Equalization Algorithm"*: `CLAHE`
+> 4. Rename {% icon galaxy-pencil %} the generated file to `input_smoothed_normalized`
+> 5. **Convert image** {% icon tool %} with the following parameters to convert the image to PNG:
+>    - {% icon param-file %} *"Input Image"*: `input_smoothed_normalized` file (output of **Histogram equalization** {% icon tool %})
+>    - *"Output data type"*: `PNG`
+> 6. Rename {% icon galaxy-pencil %} the generated file to `viz_smoothed_normalized`
+> 7. Click on the {% icon galaxy-eye %} (eye) icon next to the file name, to look at the file content and compare the result with `viz_normalized`.
```

Comment:
here you ask them to compare the two images, could you maybe write a sentence about what they should observe (maybe under the image below)?